assignment/tool/src/webapp/vm/assignment/chef_assignments_list_assignments.vm

```diff
@@ -81,11 +76,35 @@
 			</div>
 		</div>
 		#end
+		<div class="sakai-table-filterContainer">
+			<div class="row">
+				<form name="filterForm" class="col-md-12 inlineForm" method="post" action="#toolForm("AssignmentAction")">
+					<input type="hidden" name="eventSubmit_doFilter" value="filterOption" />
+					<label for="filterOption">$tlang.getString("view.label")</label>
+					<span class="skip">$tlang.getString("gen.listnavselect")</span>
+					<select id="filterOption" name="filterOption" size="1" onchange="SPNR.disableControlsAndSpin(this, null);document.filterForm.submit();">
+					#foreach($filter in $!filterOptions)
+						#set($filterMessageKey = "listassig.filter." + $filter)
+						<option value="$!filter" #if($!filter.equals($!currentFilterOption)) selected="selected" #end>$!tlang.getString($!filterMessageKey)</option>
+					#end
+					</select>
+					<input type="hidden" name="sakai_csrf_token" value="$sakai_csrf_token" />
+				</form>
+			</div>
+		</div>
+		#if ($hasAssignments)
 		<div class="sakai-table-pagerContainer">
 			#pagerPanel("pager1")
 		</div>
+		#end
 	</div>
 
+	#if (!$!hasAssignments)
+	<div class="sak-banner-info">
```

Comment:
spacing

Reply:
Do you mean I should indent the div? I don't like it so much, because it makes the markup more complex, but ill stick to how it's done now.

Reply:
Well yeah, I reckon these files sometimes are a mess, stick to the current indentation.